app/lib/spam_check.rb

```diff
@@ -88,13 +88,17 @@ def auto_silence_account!
   end
 
   def auto_report_status!
-    ReportService.new.call(Account.representative, @account, status_ids: [@status.id]) unless @account.targeted_reports.unresolved.exists?
+    ReportService.new.call(Account.representative, @account, status_ids: @status.distributable? ? [@status.id] : nil, comment: I18n.t('spam_check.spam_detected_and_silenced')) unless @account.targeted_reports.unresolved.exists?
```

Comment:
This will still silence without sending a report if a report is already opened for that account.

Reply:
Let's say someone else has reported that account for spamming already. *Not* silencing just because of that doesn't make sense to me. If you ask me, the presence of *any* report would be enough to highlight the fact that the account is silenced, since that is displayed on the reports screen.